### DISCUSSION TITLE
Microphone is muted in mozilla hubs

### DISCUSSION THREAD

**Discussion by rizwin** [mozilla/hubs#5334] [Hubs Cloud Questions]
After deploying the hubs in aws server the microphone is muted and not able to unmute it
![MicrosoftTeams-image](https://user-images.githubusercontent.com/1324951/161676072-7d0727ad-7eb6-4caf-9774-48bccfa080ce.png)

**Comment by pattersonbl2** [comment]
It is a known bug we are working to fix. We currently don't have a timeline for a completed fix.
